### DISCUSSION TITLE
Busy week 3-7 April 2023

### DISCUSSION THREAD

**Discussion by paoloserra** [caracal-pipeline/caracal#1460] [General]
# Venue

@o-smirnov please fill in details
Note: 7 April is a public holiday in RSA

# Attendees:
Present:
- Julia
- Dane
- Ruby
- Erwin
- Lexy
- Filippo
- Ben
- Athanaseus
- Oleg
- Francesca

Remote:
- Paolo

Maybe:
- Kshitij
- Peter

To be invited?
- Simon
- Jon
- Keegan
- Sphe

# Tasks

(part new, part left from previous busy week)

- RELEASE
- Stimela-2
- Add quartical to the selfcal worker? Maybe as part of a new (independent) selfcal worker?
- imcontsub may need improvements in the masking of channels (Filippo)
- SoFiA-2 implementation into CARACal with new parameters in place (Dane)
- Polarisation calibration (Francesca's top priority)
- ddcal (Franscesca's second priority, Kshitij)
- RM (Francesca's lowest priority, Lexy)
- Latest (parallelised) MosaicQueen into Stimela and Caracal (Paolo)
- Adapting the user clean mask to the user cube along the spectral axis: includes going from wide to narrow chans, from narrow to wide chans, and making a channel selection. PR open. For now just channel rebinning / selection. May need improvement later. (Julia)
- Masking worker update / better integration into pipeline (Filippo)
- Check Sharpener (Filippo)
- Add tutorial / improve docs, both schema (nesting of parameters) and other pages, selfcal worker particularly bad (Julia, Paolo, Kshitij, Francesca). See https://diataxis.fr ?
- Add installation tutorial with video (Athanaseus)
- Add CARACal/radiopadre tutorial for ilifu.
- Add header with info to the code to help further development (Lexy)
- Improve / enhance diagnostics and reports.
- https://github.com/caracal-pipeline/caracal/issues/1468 (Paolo)

**Comment by o-smirnov** [comment]
I have booked the conference facilities at Chapman's Peak Hotel, Hout Bay for 3-7 April. 

I have asked them to reserve 10 single rooms for 2-8 April. I think it's time to make a spreadsheet for the participants so that we can nail down arrival and departure dates etc.

**Comment by o-smirnov** [comment]
Please sign yourself up via the spreadsheet. Hotel bookings will be made on that basis: https://docs.google.com/spreadsheets/d/1rxkH9-t7j9ijxXlJQLsmTOHs1ZSjvb7hR4Mqg6QwQ0M/edit?usp=sharing

**Comment by o-smirnov** [comment]
One (minor) item for the busy week: fix this properly: https://github.com/caracal-pipeline/caracal/pull/1439

We can't stay pinned forever...
